docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+You can install add-ons on a Kubernetes cluster using `kubectl apply -f`.
+
+This page lists some of the available add-ons.
+
+## Networking
+
+* [Weave Net](https://github.com/weaveworks/weave-kube) is a fast, reliable pod network that carries on working even in the face of network partitions, and doesn't have any infrastructure or database dependencies. Install it on a cluster whose kubelets are configured to expect a [CNI network plugin](/docs/admin/network-plugins/):
```

Comment:
I also think eventually a table view might be good for this. It'll force us to keep the one sentence descriptions short. I would like to keep the `kubectl apply -f` copyable commands though to demonstrate how easy it is to install these add-ons to users.